terraform/loadbalancer.tf

```
@@ -0,0 +1,137 @@
+resource "aws_lb" "keyconjurer" {
+  name_prefix = "keycon"
```

I'd just use `keyconjurer` if possible :) 

```suggestion
  name_prefix = "keyconjurer"
```

---

You can't. :(

The maximum prefix length is 6.